### ISSUE TITLE
Merge staging into master

### ISSUE THREAD

**Issue by ethack** [ocmdev/rita-blacklist#11]
Would like to provide support for custom blacklists.  I'm creating this issue to track what still needs to be done before merging staging into master.

**Comment by ethack** [comment]
Superseded by rita-bl.  Rita-bl appears to contain the feature for custom blacklists as well.
